### PR TITLE
Fix style in build.gradle

### PR DESCRIPTION
**Pull request by Paul1365972**
Fixes style issues and the legacy plugin declaration

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,44 +1,34 @@
-buildscript {
```
Comment (Derongan) [inline build.gradle:1]: This is a work around for issues with how gradles plugins apply in multiproject projects. If you do not do this shadow in the parent has a fight with shadow in the submodule and everything explodes.

File: build.gradle [hunk 2]
```diff
@@ -1,44 +1,34 @@
...
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
 }
 
 repositories {
-    mavenCentral()
     mavenLocal()
+    mavenCentral()
```
Comment (Derongan) [inline build.gradle:16]: I'm not sure why moving things to be non alphabetical is a fix thats needed :P

File: build.gradle [hunk 3]
```diff
@@ -47,33 +37,36 @@ publishing {
     }
     publications {
         gpr(MavenPublication) {
-            from(components.java)
+            from components.java
+            artifact tasks.shadowJar
```
Comment (Derongan) [inline build.gradle]: We don't want to do this. If you have a dependency on geary you should not be automatically getting all of the transient dependencies as well... you just need geary's API.

**Review by Derongan — CHANGES_REQUESTED** [review]
I think this whole review is more or less a nit, but not against beautification. A few changes are changing intentional behavior though.

File: build.gradle (hunk 2 above)
Comment (Paul1365972) [reply build.gradle:16]: I prefer logical order over alphanumerical order, mavenLocal() first so all remote repositories are grouped together :p

File: build.gradle (hunk 1 above)
Comment (Paul1365972) [reply build.gradle:1]: Figured out why this happend. When including this project as a subproject the two versions of the shadow clashed (even though they are the same... anyways). The fix not being hacky with includes, but using the designated feature: [Composite Builds](https://docs.gradle.org/current/userguide/composite_builds.html)
Talked with @0ffz and will also fix this in all other plugins

File: build.gradle (hunk 3 above)
Comment (Paul1365972) [reply build.gradle]: Fixed

**Review by Derongan — APPROVED** [review]

**Review by Derongan — APPROVED** [review]

**Pull request merged**